src/front-end/src/components/MentorForm.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable */

```

Comment:
Same comment as above, we can't disable the linter on whole components, let's remove this and fix the linting errors before we merge.

Reply:
done